Modules/Operations/Entities/OfficeLocation.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Modules\operations\Entities;
```

Comment:
Please use proper naming Convention as it will break in Ununtu and Mac systems
```suggestion
namespace Modules\Operations\Entities;
```